buildkite/shared-build-env.dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM vimc/node-docker:master
+FROM vimc/node-docker:mrc-3682
```

Comment:
Remember to change before merging

Reply:
I will update other git repo using this changes before merging. I initially tried to merge changes into this docker file but it increased the build time massively, which I am not v. happy about. I have elected to update others repos.